openjdk-12.yaml

```diff
@@ -0,0 +1,219 @@
+package:
+  name: openjdk-12
+  version: 12.0.2.10
+  epoch: 0
+  description:
+  copyright:
+    - license: GPL-2.0-only
+  dependencies:
+    runtime:
+      - freetype
```

Comment:
```suggestion
      - openjdk-12-jre
      - freetype
```

Reply:
should we have the jdk depend on the jre so `$_java_home/bin/java` is available from `apk add openjdk-12`?